.github/ISSUE_TEMPLATE/implementing-new-platform.md

```diff
@@ -0,0 +1,55 @@
+# Implementing a new supported platform
+
+## At Development
+Create PR's addressing the following:
+- [ ] [coreos-assembler](https://github.com/coreos/coreos-assembler)
+    - [Example PR](https://github.com/coreos/coreos-assembler/pull/2489)
+    - [ ] `Implement required functionality to support new platform`
+- [ ] [fedora-web](https://pagure.io/fedora-web/websites)
+    - [Example PR](https://pagure.io/fedora-web/websites/pull-request/221#request_diff)
+    - [ ] `Add platform to the sites/static/js/coreos-download.js`
+- [ ] [stream-metadata-go](https://github.com/coreos/stream-metadata-go)
+    - [Example PR](https://github.com/coreos/stream-metadata-go/pull/45/)
+    - [ ] `Add platform to the Media struct in release/release.go`
+    - [ ] `Add supporting code for new platform to _toStreamArch_ func found in the release/translate.go` 
+    - [ ] (Cloud Only) `Cloud Images need to have a Images field`
+- [ ] (Cloud Only) [stream-metadata-rust](https://github.com/coreos/stream-metadata-rust/)
+    - [Example PR](https://github.com/coreos/stream-metadata-rust/pull/16)
+- [ ] [build pipeline](https://github.com/coreos/fedora-coreos-pipeline)
+    - [Example PR](https://github.com/coreos/fedora-coreos-pipeline/pull/500)
+    - [ ] `Add platform to the list found in jobs/build.Jenkinsfile for building the new artifact`
+- [ ] [fedora-coreos-browser](https://github.com/coreos/fedora-coreos-browser)
+    - [Example PR](https://github.com/coreos/fedora-coreos-browser/pull/35)
+    - [ ] `Add a list element for new platform in browser/index.html`
+- [ ] [fedora-coreos-docs](https://github.com/coreos/fedora-coreos-docs)
+    - [Example PR](https://github.com/coreos/fedora-coreos-docs/pull/377)
+    - [ ] `Add a provisioning-[platform].adoc that walks through how to setup the new platform` 
+
+## At Release
+
+1. Merge:
+    - [ ] Platform support enhancements made it to coreos-assembler
+    - [ ] Fedora website changes
+    - [ ] Stream-metadata-go changes
+    - [ ] Fedora-coreos-browser changes
+2. Wait for updates made to coreos-assembler to be propagated to latest container
+    - [ ] Download Latest version of coreos-assembler container. Verify platform support functionality. 
+3. Merge:
+    - [ ] Build pipeline changes
+4. Tag for bot pickup: [see semantic versioning](https://semver.org)
+    - [ ] Long tag stream-metadata-go commit with an appropriate new version.
+    - [ ] Long tag fedora-coreos-stream-generator commit with a new appropriate version.
```

Comment:
This can't happen until Dependabot PRs f-c-stream-generator with the new stream-metadata-go release, and the PR is merged.

Reply:
I'd restructure this a bit, so that rather than having a single `tag for bot pickup` top-level step, I'd have `release stream-metadata-go` and `release fedora-coreos-stream-generator` as separate steps, with a checkbox for running dependabot as a first step under the latter.